packages/cli/src/commands/node/install.ts

```diff
@@ -0,0 +1,45 @@
+import { BaseCommand } from "../../lib/baseCommand";
+import { ux } from "@oclif/core";
+import { ensureSwankyProject, getSwankyConfig, Spinner, downloadNode, swankyNode } from "@astar-network/swanky-core";
+import path = require("node:path");
+import { writeJSON } from "fs-extra";
+
+export class InstallNode extends BaseCommand<typeof InstallNode> {
+  static description = "Install swanky node binary";
+
+  async run(): Promise<void> {
+    await ensureSwankyProject();
+
+    const { flags } = await this.parse(InstallNode);
+    const config = await getSwankyConfig();
+    const spinner = new Spinner(flags.verbose);
```

Comment:
Base command does this, you can access them through `this.swankyConfig` and `this.spinner`

Reply:
Sorry for delayed review 😅

Reply:
No problem.

> Base command does this, you can access them through this.swankyConfig and this.spinner

That's true. thank you for the review